interp/declare.ml

```diff
@@ -231,7 +231,15 @@ let cache_variable ((sp,_),o) =
     | SectionLocalDef (de) ->
       let (de, eff) = Global.export_private_constants ~in_section:true de in
       let () = List.iter register_side_effect eff in
-      let univs = Global.push_named_def (id,de) in
+      let body = Future.chain de.const_entry_body (fun (body, ()) -> body) in
```

Comment:
Why not use Future.split2?

Reply:
Because the second value it allocates serve no purpose. Also, it's not a very good combinator for a certain FP design pattern (if I use the m-word @gares will jump around screaming).